oap-server/server-telemetry/telemetry-api/src/main/java/org/apache/skywalking/oap/server/telemetry/none/NoneTelemetryProvider.java

```diff
@@ -43,6 +44,7 @@
 
     @Override public void prepare() throws ServiceNotProvidedException, ModuleStartException {
         this.registerServiceImplementation(MetricsCreator.class, new MetricsCreatorNoop());
+        this.registerServiceImplementation(MetricsCollector.class, new MetricsCollectorNoop());
```

Comment:
Where is the change of `PrometheusTelemetryProvider`? I think this new service is also missing there, which will cause that module is unable to startup.

Reply:
From the codes, look like you are not going to abandon that module because you added a new one.

Reply:
Exactly.
You could check out the docs I added to find it out.